test/integration/api/forms/forms.js

```diff
@@ -265,7 +266,7 @@ describe('api: /projects/:id/forms (create, read, update)', () => {
           .send(testData.forms.simple2)
           .set('Content-Type', 'application/xml')
           .expect(200)
-          .then(() => asAlice.get('/v1/projects/1/forms/simple2')
+          .then(() => asAlice.get('/v1/projects/1/forms/simple2/draft')
```

Comment:
Is this the kind of API change you're referring to in the comments? That if you have a form with only a draft, accessing `/v1/projects/1/forms/simple2` wont work anymore and you'd need to access `/v1/projects/1/forms/simple2/draft`?

Reply:
I've reverted this change.